python-regression/tests/features/machine3/config.yml

```diff
@@ -1,6 +1,6 @@
 defaults: &transaction_tests_config_files
-  db: https://s3.eu-central-1.amazonaws.com/iotaledger-dbfiles/dev/Transactions_Tests_db.tar
-  db_checksum: 756237276479da4b01deaa0c1211ca65a4c8ec6f081452ea7e8153648c53bd67
+  db: https://s3.eu-central-1.amazonaws.com/iotaledger-dbfiles/dev/TransactionsTestsDb.tar
```

Comment:
Is the change to this DB documented?
Talk to @DyrellC about how to document this